logging/gateway_interceptor.go

```diff
@@ -61,15 +62,32 @@ func EnableAccountID(o *gwLogCfg) {
 	o.acctIDKeyfunc = nil
 }
 
-type sentinelKeyType int
+func WithCodeFunc(codeFunc grpc_logrus.CodeToLevel) GWLogOption {
+	return func(o *gwLogCfg) {
+		o.codeToLevel = codeFunc
+	}
+}
+
+type sentinelKeyType struct{}
 
-const sentinelKey = sentinelKeyType(0)
+var sentinelKey = sentinelKeyType{}
+
+func SentinelValueFromCtx(ctx context.Context) (value, ok bool) {
```

Comment:
I do not think we have use case when someone will need this call. In current implementation `sentinelKey` introduces coupling between `GatewayLoggingInterceptor` and `GatewayLoggingInterceptor`. I would not encourage additional coupling with other interceptors. Do you mind removing this at least until we will face a use case when this is required?

Reply:
This was actually already a direct ask from an internal consumer, otherwise I wouldn't have included it. For the exact same reason I did the coupling, they had something they want to log, but didn't want to log in both the server and gateway.